shared/components/_button.scss

```diff
@@ -15,11 +40,25 @@
   border-radius: 2rem;
   font: normal 300 13pt / 1.5 Lato, sans-serif;
   line-height: 1;
-  box-shadow: 0 0.5rem 1rem -0.5em rgba($palette-black, 0.25);
+  box-shadow: 0 0.5rem 1rem -0.5em $palette-shadow;
   box-sizing: border-box;
   outline: 0;
   margin-bottom: 0.5rem;
   background-image: radial-gradient(circle 7.5rem at 50% 20%, rgba(white, 0.125), rgba(white, 0));
+
+	&:disabled {
```

Comment:
do all :disabled buttons look the same? what about disabled & primary|alternate|affirmative…?